kamel-image/src/commonMain/kotlin/io/kamel/image/LazyPainterResource.kt

```diff
@@ -52,13 +52,21 @@ public inline fun <I : Any> lazyPainterResource(
             .build()
     }
 
+    val cachedOutput = remember(key) {

```

Comment:
@christiandeange I'm just being being picky now, but can you name it cachedResource? then looks good to me

Reply:
Done!